aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -305,6 +305,48 @@
    "source": [
     "!aviary convert_aero_table -h"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(aviary-PMC-command)=\n",
+    "### aviary convert_prop_table\n",
+    "\n",
+    "The `aviary convert_prop_table` command will convert a legacy formatted (GASP) propeller map into an Aviary formatted propeller map.\n",
+    "\n",
+    "Users must provide the path or name of an input deck, the path to the output file, and the engine format being converted.\n",
+    "If an invalid filepath is given for the input file, pre-packaged resources will be checked for input decks with a matching name.\n",
+    "\n",
+    "Notes for input decks:\n",
+    "- There are two options for Mach number: Mach number and helical Mach at 75% radius. We provide one example for each.\n",
+    "- Comments can be added using #\n",
+    "\n",
+    "\n",
+    "Example usage:\n",
+    "```\n",
+    "`aviary convert_engine -f GASP PropFan.map PropFan.prop` Convert a GASP based propeller map\n",
+    "`aviary convert_engine -f GASP KingAir90.map KingAir90.prop` Convert a GASP based propeller map\n",
```

Comment:
these should be updated to match the new filenames

Reply:
Done.